handlers/authentication.go

```diff
@@ -0,0 +1,101 @@
+package handlers
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/kiali/kiali/business"
+	"github.com/kiali/kiali/config"
+	"github.com/kiali/kiali/log"
+)
+
+type AuthInfo struct {
+	Strategy              string `json:"strategy"`
+	AuthorizationEndpoint string `json:"authorizationEndpoint,omitempty"`
+}
+
+func AuthenticationHandler(next http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		statusCode := http.StatusOK
+		conf := config.Get()
+
+		switch conf.Auth.Strategy {
+		case "openshift":
+			business, err := business.Get()
+
+			if err != nil {
+				log.Warning("Token error: ", err)
```

Comment:
Is this a token error? Seems like this error simply comes from "business.Get()" - doesn't have to do with the token stuff yet. I think this error message needs to change, otherwise it seems like it is misleading.